docs/auth.md

```diff
@@ -235,88 +235,41 @@ Note: Github deprecated basic authentication with username and password. You can
 
 ### Configuring `ssh-auth` authentication for Git
 
-This section describes how to configure an `ssh-auth` type `Secret` for use with Git. In the example below,
-before executing any `Steps` in the `Run`, Tekton creates a `~/.ssh/config` file containing the SSH key
-specified in the `Secret`. When the `Steps` execute, Tekton uses this key to retrieve `PipelineResources`
-specified in the `Run`.
-
-> :warning: **`PipelineResources` are [deprecated](deprecations.md#deprecation-table).**
->
-> Consider using replacement features instead. Read more in [documentation](migrating-v1alpha1-to-v1beta1.md#replacing-pipelineresources-with-tasks)
-> and [TEP-0074](https://github.com/tektoncd/community/blob/main/teps/0074-deprecate-pipelineresources.md).
-
-1. In `secret.yaml`, define a `Secret` that specifies your SSH private key:
-
-   ```yaml
-   apiVersion: v1
-   kind: Secret
-   metadata:
-     name: ssh-key
-     annotations:
-       tekton.dev/git-0: github.com # Described below
-   type: kubernetes.io/ssh-auth
-   stringData:
-     ssh-privatekey: <private-key>
-     # This is non-standard, but its use is encouraged to make this more secure.
-     # If it is not provided then the git server's public key will be requested
-     # when the repo is first fetched.
-     known_hosts: <known-hosts>
-   ```
-
-   In the above example, the value for `tekton.dev/git-0` specifies the URL for which Tekton will use this `Secret`,
-   as described in [Understanding credential selection](#understanding-credential-selection).
-
-1. Generate the `ssh-privatekey` value. For example:
-
-   `cat ~/.ssh/id_rsa`
-
-1. Set the value of the `known_hosts` field to the generated `ssh-privatekey` value from the previous step.
-
-1. In `serviceaccount.yaml`, associate the `Secret` with the desired `ServiceAccount`:
-
-   ```yaml
-   apiVersion: v1
-   kind: ServiceAccount
-   metadata:
-     name: build-bot
-   secrets:
-     - name: ssh-key
-   ```
-
-1. In `run.yaml`, associate the `ServiceAccount` with your `Run` by doing one of the following:
```

Comment:
thank you @OctoElla for updating the instructions!

Can we please keep this part of the doc with the new instructions?